docs/api/field_type_reference.md

```diff
@@ -1062,6 +1063,62 @@ Within this folder, images will be named like the uploaded file, suffixed with a
 - `MyImage_large.png`
 - `MyImage_rss.png`
 
+## ImageAsset Field Type
+
+ImageAsset Field Type enables storing images in independent Content items of a generic Image content type, in the media library. It makes them reusable across system.
+
+This Field Type represents an integer value.
+
+| Name      | Internal name |
+|-----------|---------------|
+|`ImageAsset`|`ezimageasset`|
+
+#### Value object
+
```

Comment:
Value object of `ezimageasset` contains the following properties:

| Property | Type  | Description|
|----------|-------|------------|
| `destinationContentId`  |  `int` | Related content id |
| `alternativeText`  |  `string` |  The alternative image text (for example "Picture of an apple."). |